integrations/mongodb_atlas/src/haystack_integrations/document_stores/mongodb_atlas/document_store.py

```diff
@@ -157,3 +156,65 @@ def delete_documents(self, document_ids: List[str]) -> None:
         if not document_ids:
             return
         self.collection.delete_many(filter={"id": {"$in": document_ids}})
+
+    def embedding_retrieval(
+        self,
+        query_embedding: np.ndarray,
```

Comment:
The argument passed in the `run` method is a list of floats. Let's pick one signature for both.

Reply:
```suggestion
        query_embedding: List[float]
```